src/types/request/v2/broker.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ *
+ * * Broker | Subaccount
+ *
+ */
+
+export interface GetSubaccountsRequest {
```

Comment:
```suggestion
export interface GetSubAccountsRequest {
```

pascal case for each interface please, same for the other interfaces/types in this file